### DISCUSSION TITLE
Can you change the colors of specific flows?

### DISCUSSION THREAD

**Discussion by DJDaveMark** [FlowmapBlue/FlowmapBlue#88] [General]
Is it possible to specify a different color for specific flows?

Everything I see in the gallery is a single color:
https://flowmap.blue/gallery

**Comment by ilyabo** [comment]
Yes, you can add a column "color" to the flows table, like in [this example](https://docs.google.com/spreadsheets/d/1Nf5LqbBJXOhBLqbS8IVLm1lQV--p7m20DP__5U2uSKw/edit#gid=229626141)

https://flowmap.blue/1Nf5LqbBJXOhBLqbS8IVLm1lQV--p7m20DP__5U2uSKw

**Reply by Cassidy-loft** [reply]
Hi! I am trying to add a "color" to the flow table except I get an error "Input must be a number between -1000000000 and 1000000000" what should I do/how do I put in the color options? Thank you

**Reply by ilyabo** [reply]
You can safely ignore this warning, it's just the column validator which isn't set up correctly in your spreadsheet.
